lib/logstash/outputs/elasticsearch/elasticsearch-template-es5x.json

```diff
@@ -20,24 +20,23 @@
           "match" : "*",
           "match_mapping_type" : "string",
           "mapping" : {
-            "type" : "string", "index" : "analyzed", "omit_norms" : true,
-            "fielddata" : { "format" : "disabled" },
+            "type" : "text", "norms" : false,
             "fields" : {
-              "raw" : {"type": "string", "index" : "not_analyzed", "ignore_above" : 256}
+              "keyword" : { "type": "keyword" }
```

Comment:
I thought the idea was to let ES run through its defaults in the case that a stringed field is being dynamically mapped